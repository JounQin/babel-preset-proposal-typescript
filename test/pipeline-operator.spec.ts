import _ from './pipeline-operator'
import { execute } from './helpers'

const proposal = 'pipeline-operator'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(_('hello', 25)).toEqual(['Hello, hello!', 57])
})
