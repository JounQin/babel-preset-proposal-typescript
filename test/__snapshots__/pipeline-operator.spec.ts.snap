// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pipeline-operator 1`] = `"Unexpected token >"`;
